app/controllers/invitations_controller.rb

```diff
@@ -59,4 +58,37 @@ def after_invite_path_for(resource)
     users_path
   end
 
+  def resource_params
+    sanitised_params = UserParameterSanitiser.new(
+      # TODO: once we've upgraded Devise and DeviseInvitable, `resource_params`
+      # hopefully won't be being called for actions like `#new` anymore and we
+      # can change the following `params.fetch(:user)` to
+      # `params.require(:user)`. See
+      # https://github.com/scambra/devise_invitable/blob/v1.1.5/app/controllers/devise/invitations_controller.rb#L10
+      # and
+      # https://github.com/plataformatec/devise/blob/v2.2/app/controllers/devise_controller.rb#L99
+      # for details :)
+      user_params: params.fetch(:user, {}),
```

Comment:
Given that the two params passed to `UserParameterSanitiser` each need substantial comments explaining the behaviour, I think it would be worth extracting them into private methods and moving the comments to those methods.